assets/css/_route_ladder.scss

```diff
@@ -36,3 +36,30 @@
     fill: $charcoal-grey;
   }
 }
+
+.m-incoming-box {
+  align-items: center;
+  background-color: transparentize($color: $white, $amount: 0.5);
```

Comment:
Does this need to be transparent? Why not an opaque gray?

Reply:
No, that's just the way it was specified but I removed it.